quickstart/support.md

```diff
@@ -31,7 +31,13 @@ Here are other community resources:
 If you are interested in what's new in each version, please refer to the following links:
 
 * [Fluentd's ChangeLog](https://github.com/fluent/fluentd/blob/master/CHANGELOG.md).
-* [td-agent's ChangeLog](https://docs.treasuredata.com/display/public/PD/The+td-agent+Change+Log)
+* [td-agent's ChangeLog](https://github.com/fluent/fluent-package-builder/blob/master/CHANGELOG-v4.md) (End of Life in Dec, 2023)
+* [fluent-package's ChangeLog](https://github.com/fluent/fluent-package-builder/blob/master/CHANGELOG.md) (Successor of td-agent package)
```

Comment:
Thanks!
Since fluent-package is newer information, how about changing the order?

```suggestion
* [fluent-package's ChangeLog](https://github.com/fluent/fluent-package-builder/blob/master/CHANGELOG.md) (Successor of td-agent package)
* [td-agent's ChangeLog](https://github.com/fluent/fluent-package-builder/blob/master/CHANGELOG-v4.md) (End of Life in Dec, 2023)
```

Reply:
Thanks!